cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,58 @@
 /// <reference types='cypress' />
 
-describe('Web Tables page', () => {
-  it('', () => {
+describe("Web Tables page visibility ", () => {
+  beforeEach(() => {
+    cy.visit("https://demoqa.com/webtables");
+  });
+
+  it("Should check the visibility of pagination", () => {
+    cy.get(".-pagination").should("be.visible");
+  });
+
+  it("Should check the visibility of Rows count selection option", () => {
+    cy.get(".-pageSizeOptions").should("be.visible");
+  });
+
+  it("Should check the visibility and functionality of Add a new worker option", () => {
+    cy.get("#addNewRecordButton").should("be.visible");
+    cy.get("#addNewRecordButton").click();
+    cy.get('input[placeholder="First Name"]').type("Michał");
+    cy.get('input[placeholder="Last Name"]').type("Anioł");
+    cy.get("#userEmail").type("michalaniol@gmail.com");
+    cy.get('input[placeholder="Age"]').type("20");
+    cy.get('input[placeholder="Salary"]').type("10000");
+    cy.get('input[placeholder="Department"]').type("Burgers");
+    cy.get("#submit").click();
+  });
+
+  it("Should test the functionality of delete worker", () => {
+    cy.get("#delete-record-1").click();
+  });
+  it("Should delete all workers", () => {
+    cy.get("#delete-all-records").click();
+    //there is no such button??
```

Comment:
Test case for deleting all workers should include assertions to verify that all workers are no longer present in the table.